### ISSUE TITLE
Add link to download dataset

### ISSUE THREAD

**Issue by jchazalon** [soduco/Benchmark_historical_map_vectorization#1]
There should be a link to download the dataset both in <https://github.com/soduco/Benchmark_historical_map_vectorization/blob/main/README.md> and <https://github.com/soduco/Benchmark_historical_map_vectorization/blob/main/dataset/README.md>.

For the current repo, only the weights of the networks are available under the [Releases tab](https://github.com/soduco/Benchmark_historical_map_vectorization/releases).

While there are some elements under the [DGMM 21 Releases page](https://github.com/soduco/paper-dgmm2021/releases) and the [ICDAR 21 Releases page](https://github.com/soduco/ICDAR-2021-Vectorization/releases), I believe there should be a direct link and the dataset should be stored both under the Releases page of the current repo, and on Zenodo.

Checklist:

- [ ] add dataset under Releases page
- [ ] check compatibility with repo code
- [ ] add dataset on Zenodo
- [ ] add link to dataset on main [README.md](https://github.com/soduco/Benchmark_historical_map_vectorization/blob/main/README.md)
- [ ] add link to dataset on dataset [README.md](https://github.com/soduco/Benchmark_historical_map_vectorization/blob/main/dataset/README.md)

**Comment by Vynikal** [comment]
Yep, this is slightly confusing. And using the full paths in the training code doesn't help either.
I'm hoping to use this benchmark for my own work. But there are absolutely no notes about the environment whatsoever. Could someone add info about python version, required packages etc.?
Thanks

**Comment by jchazalon** [comment]
Data was made available here ↓
https://datadryad.org/stash/share/V0gKnI9pWRMdzJnusQfn8HYuxCK3pbqCDGAFEf5jsf0
for the publication of this paper ↓
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0298217

@Vynikal Please let me now if the dataset is the one you actually need (I checked and believe so).
Feel also free to open another issue with specific questions about the runtime environment.
Thanks for taking the time to contribue.

**Comment by Vynikal** [comment]
Thank you for prompt response!
I downloaded the dataset from this repo:
https://github.com/soduco/ICDAR-2021-Vectorization/tree/main/dataset
Regarding the environment, I managed surprisingly fast, by using [this docker image](https://hub.docker.com/layers/nvidia/cuda/12.1.1-devel-ubuntu20.04/images/sha256-5bd13c67a4479a1c13238b470d89a92937ce68ba5f21b930d50c463e3314f657?context=explore) with Python 3.8. Then I used the requirements.txt from the aforementioned repo and with some additional dependencies, I got training.
If I have more questions, I'll open another issue.
